modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -11,7 +11,7 @@ How you create an asset depends on its type.
 
 * For OAS 2.0, RAML, RAML fragments, HTTP, WSDL, and Custom assets, create the asset directly using the Exchange *New asset* menu.
 * For an example or template, create each using the Mavenize feature in Anypoint Studio, and publish each to Exchange.
-* For a connector, see xref:to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven].
+* For a connector, example, or template, see xref:to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven].
```

Comment:
@mcampo , @LucasLudueno 
Shouldn't we also add "policy" to the list of assets that can be published to Exchange via Maven?

Reply:
Yes we should. This change is already in https://github.com/mulesoft/docs-exchange/pull/64 , and I've now ported it into this PR as well.

Reply:
yes @kuntald, since we are going to show policies in Exchange the next week